conventions.md

```diff
@@ -0,0 +1,68 @@
+# Starlark Language Conventions
+
+This document describes non-normative but generally recommended conventions for
+Starlark.
+
+## API Documentation
+
+### Docstrings
+
+API documentation for a function should be provided in a [Python-style
+docstring](https://peps.python.org/pep-0257/) - a string literal which is the
+first line of the function's body.
+
+By convention, this string should consist of an optional newline, followed by
+a 1-line, 1-sentence summary, optionally followed by a blank line and blank-line
+delimited paragraphs of additional documentation text.
+
+If a string literal is the first statement in a Starlark source file, it is
+treated as the documentation for that file; the conventions are the same as for
+a function's docstring.
+
+For example:
+
+```python
+"""A collection of useful utilities"""
```

Comment:
Nit: According to the PEP it should end in a period. It also says function/method summary lines should be written in the imperative sense. I don't know that it makes sense to inherit either of those conventions here in the language spec repo. Maybe that's the answer to my earlier question of whether or not we're bound by PEP 257.

Reply:
Added a period :)